src/web/services/search.service.ts

```diff
@@ -78,14 +80,21 @@ export class SearchService {
     const paramMap: { [key: string]: string } = {
       searchkey: searchKey,
     };
-    return this.httpRequestService.get('/search/students', paramMap);
+    return this.httpRequestService.get(ResourceEndpoints.SEARCH_STUDENTS, paramMap);
   }
 
   searchInstructors(searchKey: string): Observable<Instructors> {
     const paramMap: { [key: string]: string } = {
       searchkey: searchKey,
     };
-    return this.httpRequestService.get('/search/instructors', paramMap);
+    return this.httpRequestService.get(ResourceEndpoints.SEARCH_INSTRUCTORS, paramMap);
+  }
+
+  searchComments(searchKey: string): Observable<CommentSearchResults> {
```

Comment:
Remember to add doc.

Reply:
You mean javadoc to indicate that it's not currently functional?

Reply:
To explain what the function does (though the name might already be self-explaining)

https://typedoc.org/guides/doccomments/